vnext/Microsoft.ReactNative/Fabric/Composition/Modal/WindowsModalHostViewShadowNode.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#pragma once
+
+#include <react/renderer/components/modal/ModalHostViewState.h>
+// [Windows
+#include <react/renderer/components/rnwcore/EventEmitters.h>
+#include <react/renderer/components/rnwcore/Props.h>
```

Comment:
Had to override and rename some of the upstream Modal files to use our version of EventEmitters.h and Props.h. Without the overrides, Fabric couldn't find these files.

Reply:
Is it not possible to use the plaftform-specific view props? Don't we do this elsewhere?

Reply:
This file is just copied from upstream and is overridden to use "rnwcore/EventEmitter.h" instead of "rncore/EventEmiitter.h. Not sure why our VS code can't find the original rn-file but all our files in our repositiory reference rnwcore instead of rncore. See: https://github.com/microsoft/react-native-windows/blob/bdd2c02e56679ee64452f50407dbb3955dc4846a/vnext/Microsoft.ReactNative/Fabric/Composition/TextInput/WindowsTextInputShadowNode.h#L6